### PR TITLE
add <c-l>

### PR DESCRIPTION
**Pull request by Alok**
to clear the screen without switching to the REPL

### CODE REVIEW

File: plugin/iron.vim [hunk 1]
```diff
@@ -5,9 +5,10 @@ vnoremap <silent> <Plug>(iron-send-motion)
 
 "Call previous command again
 nnoremap <silent> <Plug>(iron-repeat-cmd) :call IronSend("\u001b\u005b\u0041")<CR>
-nnoremap <silent> <Plug>(iron-cr)         :call IronSend("")<CR>
-nnoremap <silent> <Plug>(iron-interrupt)  :call IronSend("\u0003")<CR>
-nnoremap <silent> <Plug>(iron-exit)       :call IronSend("\u0004")<CR>
+nnoremap <silent> <Plug>(iron-cr)         :call IronSend("")<CR> " <CR> to force execution of command
+nnoremap <silent> <Plug>(iron-interrupt)  :call IronSend("\u0003")<CR> " <c-c> to interrupt command
+nnoremap <silent> <Plug>(iron-exit)       :call IronSend("\u0004")<CR> " <c-d> to exit iron
+nnoremap <silent> <Plug>(iron-clear)       :call IronSend("\u000C")<CR> " <c-l> to clear screen
```
Comment (hkupty) [inline plugin/iron.vim]: ✂️ alignment

**Comment by Alok** [comment]
Done. Only took a month to remember to do this one keystroke.

**Comment by Alok** [comment]
How do you get those unicode values anyway? I forgot how I found them and I want to add another for "down" to scroll down the input.

**Comment by hkupty** [comment]
I don't remember how I exactly I logged the up command, but those are mostly ascii characters. You can find them on any simple ascii table, except for the up arrow, which is a combination of keystrokes.

**Comment by Alok** [comment]
Can this be merged in?

**Comment by hkupty** [comment]
Done. Thanks a lot!

**Pull request merged**